web/modules/custom/solution/solution.behat.inc

```diff
@@ -333,6 +335,51 @@ class SolutionSubContext extends DrupalSubContextBase implements DrupalSubContex
   }
 
   /**
+   * Subscribes the given users to the given solution.
+   *
+   * The role Id of organic group roles are consisting of 3 parts, the entity
+   * type ID and the bundle of the group, and the machine name of the role.
+   * Do not provide the complete ID of the Og role. The entity type ID and
+   * bundle are going to be added automatically.
+   *
+   * Table format:
+   * | solution          | user | roles                      |
+   * | Sample solution A | John | facilitator, administrator |
+   * | Sample solution B | Jack |                            |
+   *
+   * @param TableNode $membership_table
+   *   The membership table.
+   *
+   * @throws \Exception
+   *   Thrown when a solution is not found.
+   *
+   * @Given (the following )solution user memberships:
+   */
+  public function givenSolutionUserMemberships(TableNode $membership_table) {
+    foreach ($membership_table->getColumnsHash() as $values) {
+      // Load group.
+      $group = $this->getSolutionByName($values['solution']);
```

Comment:
Also, the getSolutionByName ends up in getRdfEntityByLabel which already handles the possibility of non existent item. You do not need to check if the group is not found.